File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/order/exception/CanNotRefundOrderException.java
package band.gosrock.domain.domains.order.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class CanNotRefundOrderException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new CanNotRefundOrderException();

    private CanNotRefundOrderException() {
        super(OrderErrorCode.ORDER_CANNOT_REFUND);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/order/exception/CanNotRefundOrderException.kt
package band.gosrock.domain.domains.order.exception

import band.gosrock.common.exception.DuDoongCodeException

class CanNotRefundOrderException private constructor() : DuDoongCodeException(OrderErrorCode.ORDER_CANNOT_REFUND) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = CanNotRefundOrderException()
    }
}
